charts/vald/values.yaml

```diff
@@ -4430,3 +4430,251 @@ initializer:
       # initializer.cassandra.secret.data -- cassandra secret data
       data:
         password: cGFzc3dvcmQ=
+
+# @schema {"name": "rebalancer", "type": "object"}
+rebalancer:
```

Comment:
Please put this section on the above of the `initializer` section.

Reply:
also, please take care about the order of the fields under this section.
Please refer the `agent` section.
https://github.com/vdaas/vald/blob/master/charts/vald/values.yaml#L2121

Reply:
fixed